department-of-veterans-affairs/caseflow
# Ki/APPEALS-54280 - Correspondence - Add tasks to newly linked appeals

Resolves [APPEALS-54280 - Correspondence and Appeal Tasks: Inbound Ops Team User: Add tasks to newly linked appeals: grey table](https://jira.devops.va.gov/browse/APPEALS-54280)

# Description
This updates the _Correspondence and Appeal Tasks_ tab on the Case Details page to have linked appeals appear in their own individual dropdown sections, including temporarily selected appeals from the list of Existing Appeals.
Note: This also resolves the last AC for [APPEALS-58114](https://jira.devops.va.gov/browse/APPEALS-58114) and unblocks testing for auto-expansion.
Note2: This also updates the wording for the TaskAdded on the Linked appeals:
1. If appeal, if has tasks: "Tasks added to appeal" and list of tasks.
2. If appeal, has no tasks: "There are no tasks on this appeal."
3. If no appeal (temporarily added): "There are no tasks on this appeal.  The linked appeal must be saved before tasks can be added"

## Acceptance Criteria

## Testing Plan
1. Navigate to the Correspondence Details page for any correspondence.
2. The existing appeals already connected to that correspondence will have linked appeals sections that are not expanded populate under the Existing Appeals section.
3. Add any prior appeals to link to the correspondence from the Existing Appeals list.  These should populate at the end of the list of Linked appeals and auto-expand on selection.
    1. Selecting and unselecting should add/remove any appeals from the list.
4. Save at bottom of page.
5. After save, temporary linked appeals are added to the list of connected appeals (this updates the appeal state, and should remove the "The linked appeal must be saved before tasks can be added." text).
